Remove WinForms/WPF BlazorWebView controls

Eilon opened the pull request:
They are now in the dotnet/maui repo: https://github.com/dotnet/maui/pull/1199

Eilon committed:
Remove WinForms/WPF BlazorWebView controls

They are now in the dotnet/maui repo.

SteveSandersonMS commented on eng/ProjectReferences.props:
```diff
@@ -150,10 +150,7 @@
-    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.WebView.WindowsForms" ProjectPath="$(RepoRoot)src\Components\WebView\Platforms\WindowsForms\src\Microsoft.AspNetCore.Components.WebView.WindowsForms.csproj" />
-    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.WebView.Wpf" ProjectPath="$(RepoRoot)src\Components\WebView\Platforms\Wpf\src\Microsoft.AspNetCore.Components.WebView.Wpf.csproj" />
-    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.WebView" ProjectPath="$(RepoRoot)src\Components\WebView\WebView\src\Microsoft.AspNetCore.Components.WebView.csproj" />
+    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.WebView" ProjectPath="$(RepoRoot)src\Components\WebView\src\Microsoft.AspNetCore.Components.WebView.csproj" />
```
Would it be OK to leave this where it was so we can still have [other platforms](https://github.com/dotnet/aspnetcore/pull/33413)?

Eilon replied:
Restored.

SteveSandersonMS reviewed (approved):
Other than my request to not move the `src` project, this all looks good!

I mean, really it actually looks awful because you're killing all our beautiful code, it's a total bloodbath - our babies! - but I'll get counselling and carry on.

Eilon commented:
@dotnet/aspnet-build  - how do I fix this build failure?

```
Found changes in 3 API baseline files
##[error]Detected modification to baseline API files. PublicAPI.Shipped.txt files should only be updated after a major release. See /docs/APIBaselines.md for more information.
error : Detected modification to baseline API files. PublicAPI.Shipped.txt files should only be updated after a major release. See /docs/APIBaselines.md for more information.
##[error]Modified API baseline files:
error : Modified API baseline files:
##[error]src/Components/WebView/Platforms/WebView2/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/WebView2/src/PublicAPI.Shipped.txt
##[error]src/Components/WebView/Platforms/WindowsForms/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/WindowsForms/src/PublicAPI.Shipped.txt
##[error]src/Components/WebView/Platforms/Wpf/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/Wpf/src/PublicAPI.Shipped.txt

Summary:

   5 error(s)

error : Detected modification to baseline API files. PublicAPI.Shipped.txt files should only be updated after a major release. See /docs/APIBaselines.md for more information.
error : Modified API baseline files:
error : src/Components/WebView/Platforms/WebView2/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/WindowsForms/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/Wpf/src/PublicAPI.Shipped.txt
```

I am deleting projects that never shipped RTM. They did ship as preview, but they are being deleted from here because they moved to another repo.

wtgodbe commented:
If the only failure is that one in code-check, one of us can just use our admin privileges to merge this for you.

Eilon commented:
@wtgodbe - thanks! I still have some work in this PR so I'll let you know when it gets to that point.

Eilon committed:
Restore previous folder structure

dougbu commented:
@Eilon please also remove the now-useless block at https://github.com/dotnet/aspnetcore/blob/main/eng/Build.props#L46-L54

TanayParikh commented on AspNetCore.sln:
```diff
@@ -1,7 +1,7 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.0.31324.12
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31320.298
```
Is this intentional?

Eilon replied:
I have unintentionally not yet installed VS17. But I will do that soon. I assume this header flip-flops all the time anyway?

Eilon committed:
Fix SLNs

Eilon committed:
Remove unused build exclusion

Eilon commented:
@SteveSandersonMS the folder structure is restored, so this is a more clean removal now.

Eilon commented:
@wtgodbe - can you super-admin merge this PR now? The code is done and it's signed off.
